source/deployment/sso-google.md

```diff
@@ -0,0 +1,39 @@
+# Google Single-Sign-On Setup (Enterprise)
+___
+
+Follow these steps to configure Mattermost to use Google as a single-sign-on (SSO) service for team creation, account creation and sign-in.
+
+1 - [Log in](https://accounts.google.com/login) to your Google account.
+2 - Go to [Google API Manager](https://console.developers.google.com), click **Credentials** in the left hand sidebar, click *Create a project** and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.
+
+![google_1_credentials](../../source/images/google_1_credentials.PNG)
```

Comment:
Most of these images aren't rendering. Possibly due to the capitalization of the "PNG" here but not in the image file name?